src/nlsolve/type.jl

```diff
@@ -17,14 +24,16 @@ mutable struct NLSolver{algType,iip,uType,tType,C<:AbstractNLSolverCache} <: Abs
   maxiters::Int
   status::NLStatus
   cache::C
+  method::MethodType
```

Comment:
Do we have plans to standardize NLSolver code? We should probably move this to DiffEqBase soon

Reply:
Not soon. We still have quite a bit of refactoring to do.